manifests/role/umich_and_ruby.pp

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2020 The Regents of the University of Michigan.
+# All Rights Reserved. Licensed according to the terms of the Revised
+# BSD License. See LICENSE.txt for details.
+
+# Frobscottle
+class nebula::role::umich_and_ruby {
```

Comment:
Thumbs up. This should eventually roll into more specific roles as they emerge.

Reply:
One could also make a datamart role that for now is just umich and ruby.

Reply:
Done.